libs/auth-api-lib/src/lib/entities/dto/delegation-scope.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { IsString } from 'class-validator'
+
+export class DelegationScopeDTO {
+  @IsString()
+  @ApiProperty()
+  delegationId!: string
+
+  @IsString()
+  @ApiProperty()
```

Comment:
`ApiPropertyOptional`

Reply:
The swagger documentation is most likely only for island.is devs, right? ... so it doesn't really matter.

Reply:
We use this for testing the service for instance